### PR TITLE
Adpone Bid Adapter: remove usersync and pass gpdr via querystring

### PR DESCRIPTION
**Pull request by seergiioo6**
## Type of change

- [ ] Bugfix
- [ x] Feature
- [ ] New bidder adapter   
- [x ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Does this change affect user-facing APIs or examples documented on http://prebid.org?
- [ ] Other

## Description of change
Remove the usersync and pass gdpr via querystring

- test parameters for validating bids
```
{
  bidder: '<bidder name>',
  params: {
    // ...
  }
}
```

Be sure to test the integration with your adserver using the [Hello World](/integrationExamples/gpt/hello_world.html) sample page.

- contact email of the adapter’s maintainer
- [ ] official adapter submission

For any changes that affect user-facing APIs or example code documented on http://prebid.org, please provide:

- A link to a PR on the docs repo at https://github.com/prebid/prebid.github.io/

### CODE REVIEW

**Comment by osazos** [comment]
Hi @seergiioo6 

Please fix your tests, they are broken.

By the way, in order to keep the context of each bidder, could you wrap all your tests under `describe('adponeBidAdapter', function () {…`? I mean:

- describe('interpretResponse'…,
- describe('getUserSyncs',
- describe('test onBidWon function',

Thanks

**Comment by ChrisHuie** [comment]
@seergiioo6 it looks like our CircleCI tests didn't run for this specific pull-request because you are currently following your fork of Prebid instead of the source Prebid project in CircleCI. Can you please unfollow your fork and follow the Prebid source project in CircleCI to make the tests automatically kick-off. Let me know if this doesn't make sense and I can further provide assistance.

**Comment by seergiioo6** [comment]
Hi,

thanks for your help.

I just unfollowed my fork in CircleCI.

Is that enough?

Thanks again

**Comment by ChrisHuie** [comment]
The CircleCI tests appear to still not have kicked off. Have you also followed the main Prebid project in CircleCI?

**Comment by seergiioo6** [comment]
Hi,

I don't know how to follow the main Prebid project in CircleCI..

I can only see my own projects in the CircleCI dashboard.

Thanks

<http://www.avg.com/email-signature?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
Libre
de virus. www.avg.com
<http://www.avg.com/email-signature?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
<#DAB4FAD8-2DD7-40BB-A1B8-4E2AA1F9FDF2>

**Comment by ChrisHuie** [comment]
You might need to switch to the Prebid organization in the top left? Prebid.js should just pop-up under the projects tab in CircleCI and you can then follow the main project.

**Comment by seergiioo6** [comment]
Hi,

I can not see the prebid organization in the top left corner dropdown.

<http://www.avg.com/email-signature?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
Libre
de virus. www.avg.com
<http://www.avg.com/email-signature?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
<#DAB4FAD8-2DD7-40BB-A1B8-4E2AA1F9FDF2>

**Comment by ChrisHuie** [comment]
Let me see what I can do to get the tests to kick off on this pr.

**Comment by ChrisHuie** [comment]
@seergiioo6 I managed to kick off the tests for CircleCI. Now the tests should automatically trigger when you add new commits to this pr. Looks like there were just a couple of linting things that popped up :)

**Comment by osazos** [comment]
Thx @ChrisHuie,
@seergiioo6 please take a look at my first comment.

**Comment by seergiioo6** [comment]
Hi,

Thank you Chris.

Yes, I'm going to wrap the tests under describe('adponeBidAdapter')
tomorrow.

Thank you very much.

**Comment by ChrisHuie** [comment]
@seergiioo6 just following up on this to make sure there weren't any other issues getting the test infrastructure to run

**Comment by osazos** [comment]
Hi @seergiioo6, are you stuck on this? Please let us know.

**Comment by seergiioo6** [comment]
Hi, I have submitted the test fix.

Sorry for the delay.

Thank you.

**Comment by osazos** [comment]
@seergiioo6, your tests fail.

```
[09:23:03] 
/home/circleci/Prebid.js/modules/adponeBidAdapter.js
  14:3  error  'getUserSyncs' is not defined                              no-undef
  90:1  error  Too many blank lines at the end of file. Max of 0 allowed  no-multiple-empty-lines

✖ 2 problems (2 errors, 0 warnings)
  1 error, 0 warnings potentially fixable with the `--fix` option.

[09:23:03] 'lint' errored after 29 s
[09:23:03] ESLintError in plugin "gulp-eslint"
Message:
    Failed with 2 errors
Details:
    domainEmitter: [object Object]
    domainThrown: false

[09:23:03] 'test' errored after 29 s
```

**Comment by osazos** [comment]
@seergiioo6, you have to submit only your changes. Actually your PR handles 86 commits…
Please fix it before I can merge.

**Comment by seergiioo6** [comment]
Hi , do you have an idea of how to do it? should I close this PR and open a new one?

**Comment by patmmccann** [comment]
merge master into your fork

**Comment by patmmccann** [comment]
you also have a linting error in your test file on line 115
  115:2  error  Parsing error: Unexpected token <<

**Comment by seergiioo6** [comment]
ok, done!

File: test/spec/modules/adponeBidAdapter_spec.js [hunk 1]
```diff
@@ -110,122 +110,102 @@ describe('adponeBidAdapter', function () {
...
+          seatbid: [
+            {
+              bid: [
+                {
+                  id: '613673EF-A07C-4486-8EE9-3FC71A7DC73D',
+                  impid: '2579e20c0bb89_0',
+                  price: 1,
+                  adm: '<html><a href="https://www.adpone.com" target="_blank"><img src ="https://placehold.it/300x250" /></a></html>',
+                  adomain: [
+                    'www.addomain.com'
```
Comment (patmmccann) [inline test/spec/modules/adponeBidAdapter_spec.js:130]: Please set bid.meta.advertiserDomains

**Comment by patmmccann** [comment]
Bid.meta.advertiserDomains is required to be in #5966 

Please add it to your js adapter, not remove adomain from your test file.

File: test/spec/modules/adponeBidAdapter_spec.js (hunk 1 above)
Comment (seergiioo6) [reply test/spec/modules/adponeBidAdapter_spec.js]: Hey, I have removed this property. Is that ok?

Comment (patmmccann) [reply test/spec/modules/adponeBidAdapter_spec.js]: Just add it to bid object. Search the repo for meta.advertiserDomains for examples how

**Comment by osazos** [comment]
hi @seergiioo6
Can you fix the last thing asked by @patmmccann to move forward?

**Comment by osazos** [comment]
Last one regarding your last commit: you use `bid.meta.adomain` in tests but `bid.adomain` in bidderAdapter.

In your tests 

```js
meta: {
  adomain: [
    'adpone.com'
  ]
},
```

In your bidderAdapter: 
```js
if (adponeBid.adomain && adponeBid.adomain.length > 0) {
  bid.meta = {};
  bid.meta.advertiserDomains = adponeBid.adomain;
}
```

**Comment by osazos** [comment]
Tests fail…

```
adponeBidAdapter
    interpretResponse
      ✗ should correctly reorder the server response
	expected { Object (id, requestId, ...) } to deeply equal { Object (id, requestId, ...) }
	AssertionError@webpack:///node_modules/assertion-error/index.js:74:1 <- test/test_index.js:26559:22
	assert@webpack:///node_modules/chai/lib/chai/assertion.js:141:1 <- test/test_index.js:58936:31
	assertEql@webpack:///node_modules/chai/lib/chai/core/assertions.js:1084:1 <- test/test_index.js:60050:16
	methodWrapper@webpack:///node_modules/chai/lib/chai/utils/addMethod.js:57:1 <- test/test_index.js:58066:30
	[native code]
	assertEqual@webpack:///node_modules/chai/lib/chai/core/assertions.js:1023:1 <- test/test_index.js:59989:15
	methodWrapper@webpack:///node_modules/chai/lib/chai/utils/addMethod.js:57:1 <- test/test_index.js:58066:30
	[native code]
	webpack:///test/spec/modules/adponeBidAdapter_spec.js:167:7 <- test/test_index.js:100329:89

```

**Review by osazos — APPROVED** [review]
LGTM

**Pull request merged**